elliepup/endpoint-management-scripts
# Feature/intune/uninstall classic teams

## Description
This pull request introduces two sets of scripts designed to remediate the presence of the classic Teams application by uninstalling both the machine-wide installer and any per-user instances of Teams. This ensures a clean removal of the classic Teams application from all managed devices.

## Changes included:

 
1. Script to detect Teams Machine-Wide Installer.
2. Script to remove Teams Machine-Wide Installer if detected by the detection script.
3. Script to detect per-user instance of Teams.
4. Script to remove per-user instance of Teams if detected by the detection script.

## Testing
Tested on devices with classic Teams present with various configurations to ensure accurate detection and remedation.

## How to Test
1. Deploy the scripts via Intune.
2. Monitor the result to verify successful remediation.

## Deployment
These scripts are to be deployed via Intune as remediation scripts. Ensure that the scripts are executed with appropriate permissions.

## Related Issues

- None
